Lib/glyphsLib/classes.py

```diff
@@ -1234,8 +1237,20 @@ def shouldWriteValueForKey(self, key):
         if key in ("xHeight", "capHeight", "ascender"):
             # Always write those values
             return True
+        if key == "name":
+            if getattr(self, key) == "Regular":
+                return False
+            return True
         return super(GSFontMaster, self).shouldWriteValueForKey(key)
 
+    @property
+    def font(self):
+        return self._font
+
+    @font.setter
+    def font(self, value):
+        self._font = value
```

Comment:
In the end, `font` is a very simple property. Maybe there is no need for a getter/setter?

Reply:
That's what I thought at first, but then I wondered about how it's possible to create a GSFontMaster that's not yet attached to a font. Loading from a Glyphs file will set the font property. Appending or inserting into the font will set the font property. But if you just do new_master = GSFontMaster() it won't have a font property at all. Not sure how common that is.

Reply:
The case you describe can work if we replace `self._font = None` with `self.font = None` in `__init__`?

Reply:
Yes, I guess that's simpler in this case.